flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/Fabric8FlinkKubeClient.java

```diff
@@ -269,14 +271,14 @@ public void eventReceived(Action action, Pod pod) {
 						callbackHandler.onDeleted(Collections.singletonList(new KubernetesPod(flinkConfig, pod)));
 						break;
 					default:
-						LOG.info("Skip handling {} event for pod {}", action, pod.getMetadata().getName());
+						LOG.debug("Ignore handling {} event for pod {}", action, pod.getMetadata().getName());
 						break;
 				}
 			}
 
 			@Override
 			public void onClose(KubernetesClientException e) {
-				LOG.error("Pods watcher onClose", e);
+				LOG.debug("The pods watcher is closing.", e);
```

Comment:
Not in the scope of this PR but why is it acceptable to omit error handling? If the connection attempts of the websocket are exhausted, we will stop watching for changes.

Reply:
Maybe @wangyang0918 can tell us why this is ok. If not then this should be properly forwarded and finally handled. This could then be another subtask of https://issues.apache.org/jira/browse/FLINK-15788.

Reply:
Then let's leave this statement on ERROR for the time being.

Reply:
@GJL It is a very nice catch. I do not think the watcher will be stopped even some exception occurs. Since the default value of `watchReconnectLimit` is -1. That means the `WatchConnectionManager` will always reconnect. The only one case is the `WatchConnectionManager#close()` by expected.

However, if the `watchReconnectLimit` is configured by users via java properties or environment, the watch may be stopped and all the changes will not be processed properly. I will create a new ticket to track this. A fatal exception will be thrown when the watcher is closed.

Reply:
https://issues.apache.org/jira/browse/FLINK-15836
The ticket has been created.